{{cookiecutter.project_slug}}/.github/workflows/expo-main.yml

```diff
@@ -3,6 +3,9 @@ on:
   push:
     branches:
       - main
+
+  workflow_dispatch:
+  
```

Comment:
Allow workflow dispatch to trigger the staging build as well, this will also mean you can build from multiple branches